### PR TITLE
Add Clockify v2.1.6

### PR DESCRIPTION
**Pull request by danyim**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, if **adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not [already refused].
- [x] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/Homebrew/homebrew-cask/pulls
[already refused]: https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues
[the correct repo]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256

### CODE REVIEW

**Comment by danyim** [comment]
I'm not sure why CI failed. Could someone shed some light?

File: Casks/clockify.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+cask 'clockify' do
+  version '2.1.6'
+  sha256 'adf4e56de1b57664b6045cd157032b0a07c7b9836273e6689632851b0f6c92fc'
+
+  # drive.google.com was verified as official when first introduced to the cask
+  url 'https://drive.google.com/uc?export=download&id=1kPzQGOveprS_JFlQJG8wd0IYh-trmj1o'
+  name 'Clockify'
+  homepage 'https://clockify.me/mac-time-tracking'
+
+  depends_on macos: '>= 10.8'
```
Comment (vitorgalvao) [inline Casks/clockify.rb]: Unnecessary. Too far behind.

File: Casks/clockify.rb [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  container nested: 'Clockify Desktop.app'
```
Comment (vitorgalvao) [inline Casks/clockify.rb]: You’re misunderstanding what this line does. What were you expecting it did?

File: Casks/clockify.rb [hunk 3]
```diff
@@ -0,0 +1,14 @@
+cask 'clockify' do
+  version '2.1.6'
+  sha256 'adf4e56de1b57664b6045cd157032b0a07c7b9836273e6689632851b0f6c92fc'
+
+  # drive.google.com was verified as official when first introduced to the cask
+  url 'https://drive.google.com/uc?export=download&id=1kPzQGOveprS_JFlQJG8wd0IYh-trmj1o'
```
Comment (vitorgalvao) [inline Casks/clockify.rb]: Where are you getting this URL from?

**Review by vitorgalvao — CHANGES_REQUESTED** [review]
Before the merge, please clarify where the Google Drive URL came from.

**Comment by danyim** [comment]
The link was from this helpdesk URL: https://clockify.me/help/apps/mac-desktop-app

But now that I navigated their site, it looks like they fixed their original app URL in this link: https://clockify.me/mac-time-tracking

which points to https://clockify.me/downloads/ClockifyDesktop_216_35.zip

File: Casks/clockify.rb (hunk 2 above)
Comment (danyim) [reply Casks/clockify.rb]: I assumed that this would extract this particular file from the compressed ZIP.

File: Casks/clockify.rb (hunk 3 above)
Comment (danyim) [reply Casks/clockify.rb]: See PR comment for full context, but in short it was from their helpdesk page: https://clockify.me/help/apps/mac-desktop-app

**Comment by danyim** [comment]
Thanks for making the changes for me! Is it good to merge now?

File: Casks/clockify.rb (hunk 1 above)
Comment (danyim) [reply Casks/clockify.rb]: I took this from another cask.

**Comment by vitorgalvao** [comment]
> In retrospect, it was probably unsafe to be linking to a random Google Drive binary without any way to verify its owner, origin, or contents.

Yes, that was the the issue.

Thank you for the explanation.

File: Casks/clockify.rb (hunk 1 above)
Comment (vitorgalvao) [reply Casks/clockify.rb]: Which cask?

**Pull request merged**